requirements.txt

```diff
@@ -1,26 +1,67 @@
+astroid==1.6.1
```

Comment:
Are all these dependencies part of your changes?

Reply:
On it